op-challenger/game/fault/trace/split/provider.go

```diff
@@ -0,0 +1,78 @@
+package split
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ethereum-optimism/optimism/op-challenger/game/fault/types"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+var (
+	GetStepDataErr = fmt.Errorf("GetStepData not supported")
```

Comment:
nit: `errors.New`

Reply:
Doesn't seem to have actually been changed. :)